### PR TITLE
Use package to process csvs

### PR DESCRIPTION
**Pull request by chiaberry**
## Associated issues

Papa, can you hear me?

https://github.com/cityofaustin/atd-moped/pull/1140

## Testing
**URL to test:** 

https://deploy-preview-1141--atd-moped-main.netlify.app/moped/projects

**Steps to test:**

Steps to test:
Click download button next to search input. Check that the csv export makes sense, a good way to quickly eyeball this is by scrolling horizontally to the end and seeing that there aren't extra columns with empty headers. In prod right now, commas that are in the status update column are being interpreted as delimiters and inserting extra columns.

Question
the file that is downloaded is named project_list_view2023-09-27blahblahtodaystime.csv Do you think we should prepend the word moped to the file name?

---
#### Ship list
- [ ] Code reviewed 
- [ ] Product manager approved
- [ ] Added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#) if applicable

### CODE REVIEW

File: moped-editor/src/components/GridTable/GridTableSearch.js [hunk 1]
```diff
@@ -123,53 +123,13 @@ const GridTableSearch = ({
...
-  };
-
   /**
    * Downloads the contents of fileContents into a file
    * @param {string} fileContents
    */
   const downloadFile = (fileContents) => {
     const exportFileName =
-      query.table + format(Date.now(), "yyyy-MM-dd'T'HH:mm:ssxxx");
+      "moped-" + query.table + format(Date.now(), "yyyy-MM-dd'T'HH:mm:ssxxx");
```
Comment (chiaberry) [inline moped-editor/src/components/GridTable/GridTableSearch.js:132]: what do yall think of adding moped to the filename?

Comment (mddilley) [reply moped-editor/src/components/GridTable/GridTableSearch.js:132]: I like it!  I could see a lot of users juggling exports from different apps and knowing where this one comes from could be helpful.

File: moped-editor/src/components/GridTable/GridTableSearch.js [hunk 2]
```diff
@@ -23,7 +23,7 @@ import GridTableNewItem from "./GridTableNewItem";
 import makeStyles from '@mui/styles/makeStyles';
 import { useLazyQuery } from "@apollo/client";
 import { format } from "date-fns";
-import { get } from "lodash";
+import Papa from 'papaparse';
```
Comment (mddilley) [inline moped-editor/src/components/GridTable/GridTableSearch.js:26]: awesome to see this library makes its way in here!

**Review by mddilley — APPROVED** [review]
Love this - let papaparse do the heavy lifting! I was able to replicate the bug in production, and I don't see the same behavior in the deploy preview. 🚀 🙌

File: moped-editor/src/components/GridTable/GridTableSearch.js (hunk 1 above)
Comment (johnclary) [reply moped-editor/src/components/GridTable/GridTableSearch.js:132]: agreed!

**Review by johnclary — APPROVED** [review]
😍 

barb

**Pull request merged**